pkg/k8s/report/report.go

```diff
@@ -65,10 +65,26 @@ type Resource struct {
 	Report types.Report `json:"-"`
 }
 
+type ConsolidatedResource struct {
```

Comment:
can we update `Resource`? I think it's a bit hard to keep 2 similar structs.

Reply:
also there are no tests for this case.

Reply:
Thanks for the feedback, I'm not sure why I assumed the `Resource` struct was used by other packages in Trivy, but it doesn't seem to be the case. I'll work on updating the `Resource` struct accordingly.

> also there are no tests for this case.
Regarding the tests, I wanted to get feedback on the approach before proceeding. I'll focus on adding the tests next.